helm/reana/values.yaml

```diff
@@ -107,6 +107,7 @@ components:
       REANA_RATELIMIT_GUEST_USER: "40 per second"
       REANA_RATELIMIT_AUTHENTICATED_USER: "40 per second"
       REANA_RATELIMIT_SLOW: "1/5 second"
+      REANA_ADMIN_QUOTA_MANAGER: ""
```

Comment:
Nomenclature: The name `REANA_ADMIN_QUOTA_MANAGER` and the subsequent help phrases such as "REANA admin quota management secret", may be misread in a way that people can read it as a managment of admin quotas (as opposed to the management of user quotas). 

Can we invent some unambiguous name for the feature, say `REANA_QUOTA_MANAGEMENT_SECRET` for consistency with `X-Quota-Management-Secret`, and call the feature in the help strings "user quota management endpoint" and such? (Here and in `r-commons` and `r-server`)